address-book-with-rails/server/Gemfile

```diff
@@ -32,10 +32,8 @@ end
 
 group :development do
   gem 'listen', '~> 3.2'
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
-  gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'mutex_m'
```

Comment:
resolves this issue:
![image](https://github.com/user-attachments/assets/5ffafba5-40e3-4b1f-91ea-ed57b6e23ab4)